src/ml_neutronics/testmain.py

```diff
@@ -38,15 +38,15 @@ def normalize_data(y_data, y_val):
 def plot_data( test_case ):
     x_cells = np.arange(1,11,1)
     
-    plt.figure(1)
+    plt.figure(1e+5+test_case)
     plt.plot(x_cells, pred_out[test_case,:10], 'r')
     plt.plot(x_cells, y_val[test_case,:10], 'b')
     plt.xlabel("cell #")
     plt.ylabel('flux')
     plt.title('fast flux')
     plt.legend(["predicted","calculated"])
 
-    plt.figure(2)
+    plt.figure(1e+5+test_case+1)
```

Comment:
overlaps with next test case
```suggestion
    plt.figure(1e+5+(test_case*2)+1)
```